README.md

```diff
@@ -1,2 +1,8 @@
 # Dockstore Dictionary
- An unofficial dictionary for Dockstore.
+ An unofficial dictionary/glossary for Dockstore. Uses Python to generate a formatted dictionary, to the extent plaintext can be considered formatted. *The content of these entries have not gone through formal review yet and there may be inaccuracies/missing content at this time.*
+
+## Usage
+ Add entries to entries.py using template in entry_template.py, making sure to keep them in alphabetical order. To generate the actual glossary, run `python3 main.py` to generate plaintext output as output.txt
```

Comment:
We can use Python to put them in alphabetical order in the output so you don't have to worry about keeping `entries.py` organized (though I understand why you would want to keep it organized).

Reply:
Yeah, eventually I'd like to have Python do it in case there is an ordering mistake in entries.py

Reply:
added: https://github.com/aofarrel/Dockstore-Dictionary/pull/1/commits/4a90d542b529499d140e14d82ee4312963bdb5d6